src/main/java/org/folio/service/mapping/convertor/SrsRecordConvertorService.java

```diff
@@ -0,0 +1,139 @@
+package org.folio.service.mapping.convertor;
+
+import static org.apache.commons.collections4.CollectionUtils.isNotEmpty;
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.lang.invoke.MethodHandles;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.commons.collections4.CollectionUtils;
+import org.folio.rest.jaxrs.model.MappingProfile;
+import org.folio.rest.jaxrs.model.RecordType;
+import org.folio.rest.jaxrs.model.Transformations;
+import org.folio.service.loader.RecordLoaderService;
+import org.folio.service.mapping.MappingService;
+import org.folio.util.OkapiConnectionParams;
+import org.marc4j.*;
+import org.marc4j.marc.Record;
+import org.marc4j.marc.VariableField;
+import org.marc4j.marc.impl.SortedMarcFactoryImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SrsRecordConvertorService extends RecordConvertor {
```

Comment:
Could you push interface RecordConverto, seems it's a reason why PR can't be built

Reply:
done, somehow my IDE wasn't showing as not pushed